companion/lib/Service/HttpApi.ts

```diff
@@ -641,4 +649,164 @@ export class ServiceHttpApi {
 			}
 		}
 	}
+
+	/**
+	 * Provides JSON appinfo
+	 *
+	 */
+	#appInfoGetJson = (_req: Express.Request, res: Express.Response): void => {
+		this.logger.debug(`Got HTTP /api/appinfo/json`)
+		res.json({
+			success: true,
+			message: 'App information retrieved successfully',
+			timestamp: getTimestampUnix(),
+			data: {
+				appinfo: this.#serviceApi.appInfo,
```

Comment:
I don't like this.  
This `appInfo` object is an internal object that has no guarantees of being at all consistent between versions.  
A majority of its contents I can't see as being of any value to a caller.

I am not against exposing some of this data, but I think it needs to be done in a more controlled fashion, with only the useful bits being exposed